src/main/java/com/thevoxelbox/voxelsniper/brush/type/RulerBrush.java

```diff
@@ -23,46 +27,56 @@ public class RulerBrush extends AbstractBrush {
     private int yOffset = DEFAULT_Y_OFFSET;
     private int zOffset = DEFAULT_Z_OFFSET;
 
-    @Override
-    public void handleCommand(String[] parameters, Snipe snipe) {
-        SnipeMessenger messenger = snipe.createMessenger();
-        String firstParameter = parameters[0];
+    @CommandMethod("")
+    public void onBrush(
+            final @NotNull Snipe snipe
+    ) {
+        super.onBrushCommand(snipe);
+    }
 
-        if (firstParameter.equalsIgnoreCase("info")) {
-            messenger.sendMessage(Caption.of("voxelsniper.brush.ruler.info"));
-        } else {
-            if (parameters.length == 1) {
-                if (firstParameter.equalsIgnoreCase("ruler")) {
-                    this.zOffset = 0;
-                    this.yOffset = 0;
-                    this.xOffset = 0;
-                    messenger.sendMessage(Caption.of("voxelsniper.brush.ruler.ruler-mode"));
-                } else {
-                    messenger.sendMessage(Caption.of("voxelsniper.error.brush.invalid-parameters"));
-                }
-            } else if (parameters.length == 3) {
-                Integer xOffset = NumericParser.parseInteger(parameters[0]);
-                Integer yOffset = NumericParser.parseInteger(parameters[1]);
-                Integer zOffset = NumericParser.parseInteger(parameters[2]);
-                this.xOffset = xOffset == null ? 0 : xOffset;
-                this.yOffset = yOffset == null ? 0 : yOffset;
-                this.zOffset = zOffset == null ? 0 : zOffset;
-                messenger.sendMessage(Caption.of("voxelsniper.brush.ruler.set-x-offset", this.xOffset));
-                messenger.sendMessage(Caption.of("voxelsniper.brush.ruler.set-y-offset", this.yOffset));
-                messenger.sendMessage(Caption.of("voxelsniper.brush.ruler.set-z-offset", this.zOffset));
-            } else {
-                messenger.sendMessage(Caption.of("voxelsniper.error.brush.invalid-parameters-length"));
-            }
-        }
+    @CommandMethod("info")
+    public void onBrushInfo(
+            final @NotNull Snipe snipe
+    ) {
+        super.onBrushInfoCommand(snipe, Caption.of("oxelsniper.brush.ruler.info"));
```

Comment:
well there is a v missing here 
![grafik](https://github.com/IntellectualSites/fastasyncvoxelsniper/assets/77929075/c374a71d-274f-46d9-8751-d57bfd4ba861)